### PR TITLE
Deploy pilot and pilotstaging

### PR DESCRIPTION
**Pull request by tkporter**
### Description

Various changes that were needed when deploying `pilot` and `pilotstaging`.
I also changed the `.gitignore` to make extra sure that no new mnemonic files can be committed.

### Tested

Deployed to both `pilot` and `pilotstaging` and tested on the wallet app

### CODE REVIEW

File: packages/verification-pool-api/src/celoEnv.ts [hunk 1]
```diff
@@ -1 +1 @@
-export const CELO_ENV = 'alfajores'
+export const CELO_ENV = 'pilotstaging'
```
Comment (cmcewen) [inline packages/verification-pool-api/src/celoEnv.ts]: @jmrossy should we gitignore this?

File: packages/verification-pool-api/contracts/Attestations.ts [hunk 2]
```diff
@@ -801,6 +801,6 @@ export default async function getInstance(web3: Web3) {
     "type": "function"
```
Comment (cmcewen) [inline packages/verification-pool-api/contracts/Attestations.ts:801]: @jmrossy could we just gitignore now that they are checked in?

File: .gitignore [hunk 3]
```diff
@@ -76,13 +76,8 @@ lerna-debug.log
 **/types/typechain/
 
 # keys
-.env.mnemonic
-.env.mnemonic.alfajores
-.env.mnemonic.alfajoresstaging
-.env.mnemonic.appintegration
-.env.mnemonic.integration
-.env.mnemonic.integrationtesting
+.env.mnemonic*
```
Comment (cmcewen) [inline .gitignore:79]: niceeeee

**Review by cmcewen — CHANGES_REQUESTED** [review]
@tkporter so from my understanding of encryption, it also uses the timestamp of when it was encrypted so you will always get a diff even if the file is the same. a little annoying, but for now i think it's worth only checking in the files you specifically wanted to change

File: packages/mobile/src/geth/network-config.ts [hunk 4]
```diff
@@ -18,4 +18,12 @@ export default {
     nodeDir: `.${Testnets.alfajores}`,
     syncMode: SYNC_MODE_ULTRALIGHT,
   },
+  [Testnets.pilot]: {
```
Comment (ashishb) [inline packages/mobile/src/geth/network-config.ts:17]: (not for this PR)

@cmcewen How about we remove `network-config.ts` completely at this point? We just need a list of known networks.

Comment (cmcewen) [reply packages/mobile/src/geth/network-config.ts:17]: sounds great 👍

**Review by cmcewen — DISMISSED** [review]

**Review by nambrot — DISMISSED** [review]

**Review by nambrot — APPROVED** [review]

**Pull request merged**